checker/src/main/java/alluxio/checker/SparkIntegrationChecker.java

```diff
@@ -0,0 +1,314 @@
+  private void printResultInfo(Status resultStatus, PrintWriter printWriter) {
+    if (mAlluxioHAMode) {
+      printWriter.println("Alluixo is running in high availbility mode.");
+      if (!mZookeeperAddress.isEmpty()) {
+        printWriter.printf("alluxio.zookeeper.address is %s.%n%n", mZookeeperAddress);
+      }
+    }
+
+    if (mSparkJobResult != null) {
+      for (Tuple2<Status, String> sjr : mSparkJobResult) {
+        switch (sjr._1()) {
+          case FAIL_TO_FIND_CLASS:
+            printWriter.printf("Spark executors of IP addresses: %s "
+                + "cannot recognize Alluxio classes.%n%n", sjr._2);
+            break;
+          case FAIL_TO_FIND_FS:
+            printWriter.printf("Spark executors of IP addresses: %s "
+                + "cannot recognize Alluxio filesystem.%n%n", sjr._2);
+            break;
+          default:
+            printWriter.printf("Spark executors of IP addresses: %s "
+                + "can recognize Alluxio filesystem.%n%n", sjr._2);
+            break;
+        }
+      }
+    }
+
+    switch (resultStatus) {
+      case FAIL_TO_FIND_CLASS:
+        printWriter.println("Please check the spark.driver.extraClassPath "
+            + "and spark.executor.extraClassPath properties in "
+            + "${SPARK_HOME}/conf/core-site.xml\n");
+        printWriter.println("For details, please refer to: "
+            + "https://www.alluxio.org/docs/master/en/Running-Spark-on-Alluxio.html\n");
+        printWriter.println("***** Integration test failed. *****");
+        break;
+      case FAIL_TO_FIND_FS:
+        printWriter.println("Please check the fs.alluxio.impl property.\n");
+        printWriter.println("For details, please refer to: "
+            + "https://www.alluxio.org/docs/master/en/Debugging-Guide.html"
+            + "#q-why-do-i-see-exceptions-like-no-filesystem-for-scheme-alluxio\n");
+        printWriter.println("***** Integration test failed. *****");
+        break;
+      case FAIL_TO_SUPPORT_HA:
+        printWriter.println("Please check the alluxio.zookeeper.address property"
+            + "in ${SPARK_HOME}/conf/core-site.xml\n");
+        printWriter.println("For details, please refer to: "
+            + "https://www.alluxio.org/docs/master/en/Running-Spark-on-Alluxio.html\n");
+        printWriter.println("***** Integration test failed. *****");
+        break;
+      default:
+        printWriter.println("***** Integration test passed. *****");
+        break;
+    }
+  }
+
+  /**
+   * Main function will be triggered via spark-submit.
+   *
+   * @param args optional argument mPartitions may be passed in
+   */
+  public static void main(String[] args) throws Exception {
+    SparkIntegrationChecker checker = new SparkIntegrationChecker();
+    JCommander jCommander = new JCommander(checker, args);
+    jCommander.setProgramName("SparkIntegrationChecker");
+
+    // Creates a file to save user-facing messages
+    File file = new File("./SparkOutputFile.txt");
```

Comment:
The final message looks like:
![image](https://user-images.githubusercontent.com/25273483/35826052-77b84566-0a6c-11e8-8d89-5ec02edfebae.png)